tests/h/sentinel_filters_test.py

```diff
@@ -1,42 +1,41 @@
-# -*- coding: utf-8 -*-
-from unittest import mock
-
 import pytest
-
 import ws4py
 
-from h.sentry.helpers import filters
-from h.sentry.helpers.event import Event
+from unittest import mock
+
+from h import sentry_filters as f
```

Comment:
I'll just import both names. If there 10 it might make more sense

Reply:
Implemented in: https://github.com/hypothesis/h/pull/5742/commits/ed685304c52faf12ac801d616fd98770c233a3f2

I've also renamed the functions to is_blah instead of filter_blah, as they don't filter anything, they just test the contents they are passed.